### DISCUSSION TITLE
Table 23, Panel 1

### DISCUSSION THREAD

**Discussion by horsburgh** [cssi-pi-community/2022-meeting#17] [Panel 1 roundtable (Monday morning)]
# Panel 1 Discussion for Table 23

Ensuring a Diverse Pipeline of CI Researchers and Professionals

**Comment by horsburgh** [comment]
In the deomographic discussions about diversity it mostly comes to gender, but there is a perception that this should also be about race and other aspects that we are not tracking.

**Comment by horsburgh** [comment]
In geoscience, there is a paper from a few years ago. THe gender gap is close to closing, but the racial diversity has hardly budged in 30 years of data.

**Comment by timm** [comment]
The term “intersectionality” or “intersectional theory” was coined in 1989 [by Kimberle Crenshaw in her paper](https://chicagounbound.uchicago.edu/cgi/viewcontent.cgi?article=1052&context=uclf), “Demarginalizing the Intersection of Race and Sex: A Black Feminist Critique of Antidiscrimination Doctrine, Feminist Theory and Antiracist Politics.”

**Comment by timm** [comment]
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/29195/180831761-26541595-008d-493b-ba0e-ae273a5f40d1.png">

**Comment by horsburgh** [comment]
Engaging people from MSI's by sending Research Software Engineer to do a workshop/training program. MSI faculty are often so overworked that they struggle to do more.

**Comment by horsburgh** [comment]
Two big challenges - 1) how do we get the people that we need to support our software/science/CI development hired and paid, and 2) how do we address the DEI issues in this space. Do we address these problems together, or do we address them separately? Which comes first?

**Comment by timm** [comment]
issues for managers:
1. career path planning for the future
2.  creating safe spaces for current work (creating safe space: where people can bring their full and authentic selves)

**Reply by timm** [reply]
and is this a retention mechanism? creating a good safe space?

**Comment by horsburgh** [comment]
Creating a safe space where people can be their full and authentic self is a retention measure

**Comment by horsburgh** [comment]
Need a mechanism for soft money people to participate in proposal writing activities. There are implications of time and effort reporting that have to be addressed if soft money people are going to be participating in proposal development.

**Comment by horsburgh** [comment]
University Human Resources departments are not well set up to support our recruitment efforts for the types of professionals we need. University HR knows how to support faculty searches and day-to-day university staff, but they don't know how/where to help recruit good people - if they are wiling to help at all.

**Comment by namurphy** [comment]
Require psychological safety training and cultural competency training for PIs?
